.github/workflows/testing.yml

```
@@ -208,14 +208,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: macos-10.15
+          - os: [[self-hosted, macos-10.15], macos-latest]
             python: 3.7
             with_cc: OFF
             with_cpp_tests: ON
             wanted_archs: 'cpu'
```

```suggestion
          - os: [self-hosted, macos-10.15]
            python: 3.7
            with_cc: OFF
            with_cpp_tests: ON
            wanted_archs: 'cpu'
          - os: macos-latest
            python: 3.7
            with_cc: OFF
            with_cpp_tests: ON
            wanted_archs: 'cpu'
```

Could you please try this?
It might not work though since github hosted runners may not have brew.